openstack/tools/func_test_tools/identify_charm_func_tests.py

```diff
@@ -22,14 +22,21 @@ def extract_targets(bundle_list):
     extracted = []
     for item in bundle_list or []:
         if isinstance(item, dict):
-            extracted.append(list(item.values())[0])
+            values = list(item.values())
+            if isinstance(values, list):
```

Comment:
Will this ever be false?

Reply:
yeah this is clearly wrong i need to fix it